### ISSUE TITLE
Test Groovy Functions

### ISSUE THREAD

**Issue by Emiller88** [nf-core/demultiplex#74]
### Description of feature

- [ ] extract_csv
- [ ] parse_flowcell_csv
- [ ] ?

**Comment by matthdsm** [comment]
Updated with some groovy functions from the subworkflows, although strictly speaking, those should live in the modules repo

**Comment by Emiller88** [comment]
Yeah definitely, we can start them here and migrate them over when we move modules to nf-test
